gazebo_plugins/CMakeLists.txt

```diff
@@ -46,6 +65,11 @@ ament_package()
 install(DIRECTORY include/
         DESTINATION include)
 
+install(TARGETS gazebo_ros_force
```

Comment:
We can just have one install target for all the plugins

```
install(TARGETS
  gazebo_ros_force
  gazebo_ros_template
  ARCHIVE DESTINATION lib
  LIBRARY DESTINATION lib
  RUNTIME DESTINATION bin)

Reply:
Good call, done https://github.com/ros-simulation/gazebo_ros_pkgs/pull/784/commits/0cc0e06f4aaca8b9cabd025dde907402afc9fc7a